src/interfaces/network.ts

```diff
@@ -36,43 +34,44 @@ export type NetworkInfoLoading<T> = {
   [K in keyof T]: T[K] | 'LOADING'
 }
 
-export type NetworkFeature = {
+export interface NetworkFeature {
   id: string
   title: string
   msg?: string
   level: 'success' | 'danger' | 'warning' | 'loading' | 'initial'
 }
-
-// NetworkId is a string: this is our internal identifier for the network
-// chainId is a number and is the chainID used for replay protection (EIP-155)
-// we need this distinction because:
-// 1) it's easier to work with the string identifier, for example if we have an object segmented by networks it's easier to debug with string IDs
-// 2) multiple distinct networks may (rarely) run the same chainId
-export interface NetworkDescriptor {
```

Comment:
After the changes, this comment was deleted, but IMO it's still relevant, so I'd opt in to keep it somewhere next to the network and chain id fields.